### DISCUSSION TITLE
Error after update 7.3.0. This parameter set requires WSMan, and no supported WSMan client library was found

### DISCUSSION THREAD

**Discussion by aanikolaeff** [PowerShell/PowerShell#18499] [Q&A]
Today powershell updated by brew to 7.3.0 version. Try to use EnterPS-Session compname and got error

`This parameter set requires WSMan, and no supported WSMan client library was found. WSMan is either not installed or unavailable for this system.`

But WSMan module is installed 

`Manifest  2.3.0  PSWSMan  Core`

In version 7.2.0 all works normal.

**Comment by jborean93** [comment]
Even if you have the PSWSMan module installed every update is going to overwrite the libmi and libpsrpclient files in the pwsh update. You essentially need to run `Install-WSMan` again after updating PowerShell to replace those libs with its own.
